scripts/postlink.js

```diff
@@ -383,7 +386,8 @@ function addSentryInit() {
 function resolveSentryCliBinaryPath(props) {
   return new Promise(function(resolve, reject) {
     try {
-      props['cli/executable'] = require.resolve('sentry-cli-binary/bin/sentry-cli');
+      const cliPath = require.resolve('sentry-cli-binary/bin/sentry-cli');
+      props['cli/executable'] = path.relative(process.cwd(), cliPath);
```

Comment:
But the file is absolutized at load time in sentry-cli. So not sure why the path would mismatch later.